File: UI/UIEKrushiStdApp/src/app/Models/shipper-order.ts
export class ShipperOrder {
    constructor(
        public orderId: number,
        public fromAddressId: number,
        public toAddressId: number,
        public status: string,
        public fromAddress: string,
        public toAddress: string,
        public orderDate: string,
        public shippedDate: string
    ) {}
  }
  

File: Dotnet/NewAPI/Shippers/Models/ShipperOrder.cs
namespace TransFlower.EKrushi.Shippers.Models;

public class ShipperOrder
{
    public int OrderId { get; set; }
    public int FromAddressId { get; set; }
    public int ToAddressId { get; set; }
    public string? Status { get; set; }
    public DateTime OrderDate { get; set; }
    public DateTime ShippedDate { get; set; }
}
